linux/jdk/redhat/src/main/packaging/temurin/11/temurin-11-jdk.spec

```diff
@@ -262,6 +262,8 @@ fi
 /usr/lib/tmpfiles.d/%{name}.conf
 
 %changelog
+* Wed Feb 22 2023 Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org> 11.0.18.0.0.10.adopt0
```

Comment:
The changelog version should have the full release string which would include the `spec_release`, for example `11.0.18.0.0.8-1`. I'm not sure why there is adopt0 appended to the changelog entries, it doesn't seem to be used anywhere else.

Reply:
Thanks @jie,  I'll amend the version strings, but leave adopt0 in place for the moment!